frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx

```diff
@@ -576,6 +576,9 @@ export const CreateVMWizardPageComponent: React.FC<CreateVMWizardPageComponentPr
     );
   }
 
+  const userMode = new URLSearchParams(search).get('mode');
+  const userTemplateName = new URLSearchParams(search).get('template');
```

Comment:
we should make a null of this if the mode is something different than the `template`